snuba/migrations/snuba_migrations/replays/0001_replays.py

```diff
@@ -38,8 +38,8 @@
     Column("ip_address_v4", IPv4(Modifiers(nullable=True))),
     Column("ip_address_v6", IPv6(Modifiers(nullable=True))),
     # user columns
-    Column("user", String()),
-    Column("user_hash", UInt(64)),
+    Column("user", String(Modifiers(default="''"))),
+    Column("user_hash", UInt(64, Modifiers(materialized="cityHash64(user)"))),
```

Comment:
i'm actually not sure if we need `user_hash` -- what's the purpose on our other tables?

Reply:
As far as I can tell, we never even query this column in other datasets. So I think we could omit it.

Reply:
If you don't have a specific use case to use the column right now then I would recommend not adding it. There would be an unnecessary materialization which would have to be computed during inserts.